ui/assets/tad.js

```diff
@@ -0,0 +1,77 @@
+let shapChart;
+
+/**
+ * Good enough function to toggle tabs
+ * @param myId
+ * @param tabEl
+ */
+function toggle(myId, tabEl) {
+
+    let tabs = document.getElementsByClassName("tabs__list")[0].getElementsByTagName("li");
+    for (i =0; i < tabs.length; i++) {
```

Comment:
please declare i, else it is assumed to be a global variable

Reply:
It is better to declare them, so I did. However, in javascript variables are (very) scoped. So in this case, the variable 'i' does not exist outside the for loop. I noticed in Python variables are 'all over the place', but in javascript that is not the case (I checked) ;)